audit_events/v1/audit_events_suite_test.go

```diff
@@ -0,0 +1,105 @@
+package audit_events
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"github.com/google/uuid"
+	"log"
+	"testing"
+
+	"github.com/cloudfoundry/cf-test-helpers/v2/workflowhelpers"
+	. "github.com/onsi/ginkgo/v2"
+	"github.com/onsi/ginkgo/v2/types"
+	. "github.com/onsi/gomega"
+
+	"github.com/cloudfoundry/cf-performance-tests/helpers"
+)
+
+var testConfig = helpers.NewConfig()
+var prefix string
+var testSetup *workflowhelpers.ReproducibleTestSuiteSetup
+var ccdb *sql.DB
+var uaadb *sql.DB
+var ctx context.Context
+
+const (
+	// main test parameters:
+	orgs   = 4
+	spaces = 1
+)
+
+var _ = BeforeSuite(func() {
+
+	testSetup = workflowhelpers.NewTestSuiteSetup(&testConfig)
+	testSetup.Setup()
+	prefix = testConfig.GetNamePrefix()
+	ccdb, uaadb, ctx = helpers.OpenDbConnections(testConfig)
+	helpers.ImportStoredProcedures(ccdb, ctx, testConfig)
+
+	// create orgs
+	createOrgStatement := fmt.Sprintf("create_orgs(%d)", orgs)
+	helpers.ExecuteStoredProcedure(ccdb, ctx, createOrgStatement, testConfig)
+
+	//create spaces
+	createSpacesStatement := fmt.Sprintf("create_spaces(%d)", spaces)
+	helpers.ExecuteStoredProcedure(ccdb, ctx, createSpacesStatement, testConfig)
+
+	// create events helper table
+	createEventsTableStatement := fmt.Sprintf("create_table_event_types()")
+	helpers.ExecuteStoredProcedure(ccdb, ctx, createEventsTableStatement, testConfig)
+
+	// create events
+	createEventStatement := fmt.Sprintf("create_events()")
+	helpers.ExecuteStoredProcedure(ccdb, ctx, createEventStatement, testConfig)
+
+	// create apps table entries
+	createApps(2)
+
+	// assign the regular user to all orgs
+	regularUserGUID := helpers.GetUserGUID(testSetup.RegularUserContext(), testConfig)
+	orgsAssignedToRegularUser := orgs
+	assignUserAsOrgManager := fmt.Sprintf("assign_user_as_org_manager('%s', %d)", regularUserGUID, orgsAssignedToRegularUser)
+	helpers.ExecuteStoredProcedure(ccdb, ctx, assignUserAsOrgManager, testConfig)
+
+	helpers.AnalyzeDB(ccdb, ctx, testConfig)
+})
+
+func createApps(numApps int) {
+	i := 1
+	for i < numApps {
+		i++
+		appGuid := uuid.NewString()
+		appName := fmt.Sprintf("%s-space-%s", prefix, appGuid)
+		createSpaceStatement := fmt.Sprintf(
+			"INSERT INTO apps (id,guid, name) VALUES ('%d','%s', '%s')",
+			i, appGuid, appName)
+		helpers.ExecuteInsertStatement(ccdb, ctx, createSpaceStatement, testConfig)
+	}
+}
+
+var _ = AfterSuite(func() {
+	helpers.CleanupTestData(ccdb, uaadb, ctx, testConfig)
```

Comment:
Does `CleanupTestData` clean up the new test data?

Reply:
At least for postgres, will check for mysql again

Reply:
Yes, it does clean up the test data